packages/terra-core-docs/src/terra-dev-site/doc/icon/IconCollectionTable/IconCollectionRow.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconCheckmark from 'terra-icon/lib/icon/IconCheckmark';
+import styles from './IconCollectionRow.module.scss';
+import IconCollectionImportsCell from './IconCollectionImportsCell';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The terra-icon this row describes
+   */
+  iconSvg: PropTypes.elementType,
+  /**
+   * Physical description of icon
+   */
+  description: PropTypes.string.isRequired,
+  /**
+   * The common or intended meaning/s of the icon
+   */
+  meaning: PropTypes.string,
+  /**
+   * Whether the icon can be used as an action
+   */
+  isAction: PropTypes.bool,
+  /**
+   * Whether the icon can be used as a status indicator
+   */
+  isStatus: PropTypes.bool,
+  /**
+   * Whether the icon can be used as a toggle
+   */
+  isToggle: PropTypes.bool,
+  /**
+   * A list of sets/categories the icon belongs to
+   */
+  sets: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * A list of aliases for the icon in terra
+   */
+  importNames: PropTypes.arrayOf(PropTypes.string.isRequired),
```

Comment:
If the icon table allows to show multiple exports for the same icon, this makes it appear that all these exports reference to the same icon with the same characteristics and features, just different export name. 

Using this as an example, the table layout appears to suggest that `IconSuccess`, `IconAvailable`, and `IconComplete` are identical and can be used interchangeably. Are there any differences among these three icons? If there are, should they be on their own separate row and in this case the`importNames`  prop should be a string instead of an array of strings?

<img width="966" alt="Screen Shot 2022-02-24 at 3 43 58 PM" src="https://user-images.githubusercontent.com/22618655/155613067-813ff786-349f-4517-8a38-cb649d1a6ad2.png">

Reply:
This is intentional and the reason I chose it as an example. The icons that each `terra-icon` uses are defined by and generated from [this CSV](https://github.com/cerner/one-cerner-style-icons/blob/main/src/CernerOneIcons-CSV.csv). `IconSuccess`, `IconAvailable`, and `IconComplete` do pull from the exact same SVG file (lines 5, 9, & 10). 

Currently, the only consideration affecting which alias to use is what would make the most sense in the import statement of the code where it is used.

Reply:
Thanks for the explanation!